cfpb/cfgov-refresh
# Housing counselor: Adds aria-label to OMB control number

## Additions

- Adds aria-label for OMB control number link in the privacy act statement.

## Testing

1. Visit http://localhost:8000/find-a-housing-counselor/?zipcode=05201
2. Turn voice over on with command+f5
3. Click in the privacy act statement and tab to the "3170-0025" link and see that it reads the link in context with the surrounding sentence.

## Screenshots

<img width="582" alt="screen shot 2018-11-23 at 1 57 29 pm" src="https://user-images.githubusercontent.com/704760/48957681-1bdd0500-ef28-11e8-8d62-9ee998a77714.png">
